src/components/Tip.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Label } from '../components';
+
+const Tip = ({
+  tipText,
+  position = 'right center',
+}: {
+  tipText: string;
+  position?: string;
+}): any => {
+  return (
+    <React.Fragment>
+      <Label circular basic data-tooltip={tipText} data-position={position}>
+        ?
+      </Label>
+    </React.Fragment>
+  );
+};
+export default Tip;
```

Comment:
Nice and handy component!

Reply:
Thank you. Will be useful in the forked version too. Intention is to get rid of instruction manuals.